manual/en-US/coding-standards/chapters/php.md

```diff
@@ -131,6 +131,25 @@ switch ($value)
 }
 ```
 
+## Mixed language usage (e.g. at the layout files)
+
+For layout files and all files where we use a mix of PHP and HTML (all PHP files in the `view/tmpl` and `layout` folder) we additionally wrap every line into a `<?php ... ?>` block and use the alternative syntax for control structures.
+This should make the code easier to read. You can also easier move blocks around without creating fatal errors due to missing `<?php ... ?>` tags and the like.
```

Comment:
fixed both with: https://github.com/zero-24/coding-standards/commit/b26f6dd36ed3f30b8fb7b16a8ce2ce1d09500f6b and https://github.com/zero-24/coding-standards/commit/a28bcc56646a7d6a9eb63112dc703232b7fe7eeb